Classes/Hooks/DataHandler.php

```diff
@@ -80,6 +80,7 @@ public function processDatamap_postProcessFieldArray($status, $table, $id, &$fie
                     }
                     break;
                     // Field post-processing for table "tx_dlf_metadata".
+                    // TODO: Include also subentries if available.
```

Comment:
Are these TODOs still relevant? The two-level metadata feature should be implemented completely. So either these TODOs are done (and should be removed) or should be done.